### PR TITLE
cdc: fix typos

### PR DESCRIPTION
**Pull request by hi-rustin**
Signed-off-by: hi-rustin <rustin.liu@gmail.com>

### What problem does this PR solve?
None

Problem Summary:

### What is changed and how it works?

What's Changed:
Fix typos.

### Related changes

None

### Check List 

Tests 

- No code

Side effects

None

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by hi-rustin** [comment]
/cc @overvenus 

Could you please take a look? Thanks!

File: components/cdc/src/service.rs [hunk 1]
```diff
@@ -180,7 +180,7 @@ impl Conn {
         // Assume batch resolved ts will be release in v4.0.7
         // For easy of testing (nightly CI), we lower the gate to v4.0.6
         // TODO bump the version when cherry pick to release branch.
-        let v407_bacth_resoled_ts = semver::Version::new(4, 0, 6);
+        let v407_batch_resoled_ts = semver::Version::new(4, 0, 6);
```
Comment (sticnarf) [inline components/cdc/src/service.rs:183]:
```suggestion
        let v407_batch_resolved_ts = semver::Version::new(4, 0, 6);
```

File: components/cdc/src/service.rs [hunk 2]
```diff
@@ -198,7 +198,7 @@ impl Conn {
             }
             None => {
                 let mut features = FeatureGate::empty();
-                if v407_bacth_resoled_ts <= ver {
+                if v407_batch_resoled_ts <= ver {
```
Comment (sticnarf) [inline components/cdc/src/service.rs:201]: ditto

Comment (hi-rustin) [reply components/cdc/src/service.rs]: Fixed. Thanks for your review.

**Review by overvenus — APPROVED** [review]

**Review by ben1009 — APPROVED** [review]

**Comment by hi-rustin** [comment]
@sticnarf Ping~

**Review by sticnarf — APPROVED** [review]

**Pull request merged**